src/main/protobuf/bsp.proto

```diff
@@ -135,17 +150,43 @@ message ScalaBuildTarget {
     string scalaOrganization = 1;
     string scalaVersion = 2;
     string scalaBinaryVersion = 3;
+    ScalaPlatform platform = 4;
```

Comment:
Seems off-sync with the spec. It's tedious to keep the two synchronized but I think it's worth the effort.